web/src/drafts.js

```diff
@@ -657,6 +687,38 @@ export function launch() {
     setup_event_handlers();
 }
 
+function update_bulk_delete_ui() {
+    const $unchecked_checkboxes = $(".mark-draft").filter(function () {
+        return !is_checkbox_icon_checked($(this));
+    });
+    const $checked_checkboxes = $(".mark-draft").filter(function () {
+        return is_checkbox_icon_checked($(this));
+    });
+    const $select_drafts_button = $(".select-drafts-button");
+    const $select_state_indicator = $(".select-drafts-button .select-state-indicator");
+    const $delete_selected_drafts_button = $(".delete-selected-drafts-button");
+
+    if ($checked_checkboxes.length > 0) {
+        $delete_selected_drafts_button.prop("disabled", false);
+        if ($unchecked_checkboxes.length === 0) {
+            toggle_checkbox_icon_state($select_state_indicator, true);
+            $select_drafts_button.attr("aria-checked", "true");
```

Comment:
Yes, you are right. This can be done by modifying the `aria-checked` property of the `parent` of the checkbox that is passed to `toggle_checkbox_icon_state`.